packages/cli-repl/src/log-writer.ts

```diff
@@ -131,10 +157,14 @@ export class MongoLogWriter extends Writable {
     this._target.end(callback);
   }
 
+  /** Wait until all pending data has been written to the underlying stream. */
   async flush(): Promise<void> {
     await new Promise(resolve => this._target.write('', resolve));
   }
 
+  /**
```

Comment:
I'm trying to infer the style here. Sometimes you do `/** */` on one line and sometimes split over three even when it would have fit on one line. Is there some rule I'm missing? Maybe I'm trying to read too deeply into it :)

Reply:
> Maybe I'm trying to read too deeply into it :)

Yes :) I honestly don’t think it matters, but I’ve tried to at least be mostly consistent for members of the same class/type.